### ISSUE TITLE
<u> tags on link are an accessibility issue

### ISSUE THREAD

**Issue by EdSmilovici** [3ilogiclms/moodle-block_course_status_tracker#9]
The <u> tags wrapping anchor tags do not add any value.
The general use of <u> tags is an accessibility issue i'm currently dealing with.

I think these should be removed even if the accessibility warning they cause is a red herring because they add no value.
